pages/[pdfId].jsx

```diff
@@ -39,6 +39,23 @@ class Reader extends React.Component {
         <Head>
           <title>{title} - CORE Reader</title>
 
+          {['64', '128', '256', '512'].map(size => (
+            <link
+              rel="icon"
+              type="image/png"
+              sizes={`${size}x${size}`}
+              href={`/static/favicon/favicon-${size}px.png`}
+            />
+          ))}
+
+          <link
+            rel="icon"
+            sizes="any"
+            type="image/svg+xml"
+            href="/static/favicon/favicon.svg"
+          />
+
+          <meta name="referrer" content="origin" />
```

Comment:
Could you please explain why did you add exactly this value?

I have read the article but did not completely get your opinion about.

Reply:
It just adds referrer to request if I understand it correctly. For example, when you click on some link in PDF and that link redirects you to an external page then the owner of that external page can see that traffic comes from core.ac.uk. (It's mostly for Google Analytics. Instead of the unknown referrer, you see our webpage)